packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/representations/RepresentationIdBuilder.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.services.representations;
```

Comment:
It should be in `org.eclipse.sirius.web.tests.services.representation` to let others uses it